bin/propolis-server/src/lib/server.rs

```diff
@@ -173,6 +174,21 @@ impl VmControllerState {
     }
 }
 
+/// Objects related to Propolis's Oximeter metric production.
+pub struct OximeterState {
+    /// A task spawned at initial startup to register with Nexus for metric
+    /// production.
```

Comment:
it would be nice if this doc comment explained the circumstances under which the `Notify` is woken?

Reply:
Done in 7c4cbe7